MediaRecorder.html

```diff
@@ -80,6 +80,7 @@
     void           resume ();
     void           requestData ();
     static boolean isTypeSupported (DOMString type);
```

Comment:
See below regarding MIME types and codec strings. Unless implementations support encoding many profiles and levels, it will be difficult for applications to find exact matches. For example, `avc1.42E01E` vs. `avc1.4D401E`. For other such methods, the application has a stream and wants to know if the user agent supports it. In this case, the application must try to guess at the strings the user agent might support (or be strict in only support specific profiles and levels, which could limit client compatibility).

More generally, the spec should be more specific about checking `type`, including generally requiring `codecs`, checking audio vs. video types, and determining whether multiple codecs (e.g. audio and video, if applicable) are supported together. See, for example, how EME [handles MIME content type strings](https://w3c.github.io/encrypted-media/#get-supported-capabilities-for-audio-video-type).

Three of the four types in Example 7.1 are incorrect. Two only specify the container, and another uses `h264` rather than `avc1.xxxxxx`. Also, these are content types, not "codecs," which is the variable name.